src/routes/MainRouter.tsx

```diff
@@ -51,6 +52,7 @@ export const MainRouter: React.FC = () => {
             element={<LaboratoryActivity />}
           />
           <Route path={`${PATHS.patients}/*`} element={<PatientsRoutes />} />
+          <Route path={`${PATHS.admin}/*`} element={<AdminRoutes />} />

```

Comment:
Should we render a "403 Error component" when an admin route is called without the proper permissions?